src/Phinx/Console/Command/SeedRun.php

```diff
@@ -104,7 +105,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         // run the seed(ers)
         $start = microtime(true);
-        $this->getManager()->seed($environment, $seed);
+        foreach($seedSet as $seed) {
+            $this->getManager()->seed($environment, trim($seed));
```

Comment:
Yeah, simple `seed:run`. The bug is in `0.5.x-dev` branch, not in `master`. Follow your changes to see how it works when `$seed` is `null`. See https://github.com/robmorgan/phinx/issues/838 and [that patch](https://github.com/robmorgan/phinx/pull/835/files) for reference.